Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -599,7 +596,7 @@ private void setAttributesForProcess() {
     }
 
     /**
-     * prüfen, ob per Parameter vielleicht zunÃ¤chst ein anderer geladen werden
+     * prüfen, ob per Parameter vielleicht zunächst ein anderer geladen werden
```

Comment:
On fixing the German umlaut issue maybe you can translate this JavaDoc into English?